core/python3Action/lib/prelauncher.py

```diff
@@ -97,6 +97,30 @@ def cannot_start(msg):
 except Exception as ex:
   cannot_start("Invalid action: %s\n" % str(ex))
 
+class Context:
+  def __init__(self, env):
+    self.function_name = env["__OW_ACTION_NAME"]
+    self.function_version = env["__OW_ACTION_VERSION"]
+    self.activation_id = env["__OW_ACTIVATION_ID"]
+    self.request_id = env["__OW_TRANSACTION_ID"]
+    self.deadline = int(env["__OW_DEADLINE"])
+    self.api_host = env["__OW_API_HOST"]
+    self.api_key = env.get("__OW_AUTH_KEY", "")
```

Comment:
@rabbah and I discussed this and decided to rename `auth_key` to `api_key` to build a nice dual with `api_host`, for which it'll supposed to be used.

Reply:
As discussed in person with @markusthoemmes today, agree with this.